plugin/rewrite/name_test.go

```diff
@@ -31,3 +31,58 @@ func TestRewriteIllegalName(t *testing.T) {
 		t.Errorf("Expected invalid name, got %s", err.Error())
 	}
 }
+
+func TestNewNameRule(t *testing.T) {
+	tests := []struct {
+		next         string
+		args         []string
+		expectedFail bool
+	}{
+		{"stop", []string{"exact", "srv3.coredns.rocks", "srv4.coredns.rocks"}, false},
+		{"stop", []string{"srv1.coredns.rocks", "srv2.coredns.rocks"}, false},
+		{"stop", []string{"suffix", "coredns.rocks", "coredns.rocks."}, false},
+		{"stop", []string{"suffix", "coredns.rocks.", "coredns.rocks"}, false},
+		{"stop", []string{"suffix", "coredns.rocks.", "coredns.rocks."}, false},
+		{"stop", []string{"regex", "srv1.coredns.rocks", "10"}, false},
```

Comment:
This test, and the following one replace a name with "10" ... seems unusual.  Does this test a special case/part of code?  Was it copied from a TTL test case?

Reply:
> Does this test a special case/part of code?

@chrisohaver, No.

> Was it copied from a TTL test case?

No.

I think I just wanted to check for digits only rewrite.